exonum-light-client/README.md

```diff
@@ -15,6 +15,8 @@ The following table shows versions compatibility:
 | Light Client | Exonum | Exonum Java |
 |--------------|--------|-------------|
 | 0.1.0        | 0.10.* | 0.4         |
+_Be noticed, that the table above includes release versions only
+and does not include snapshot versions_
```

Comment:
I'd leave this out as snapshots are internal things. Or are you trying to say "Light client is tested against release versions, compatibility with unreleased versions is not guaranteed"?